src/internal/connector/exchange/service_iterators.go

```diff
@@ -131,31 +130,30 @@ func FilterContainersAndFillCollections(
 			statusUpdater,
 			ctrlOpts,
 		)
-		collections[cID] = &edc
-
-		if edc.State() == data.DeletedState {
-			continue
-		}
 
+		collections[cID] = &edc
 		edc.jobs = append(edc.jobs, jobs...)
 
 		// add the current path for the container ID to be used in the next backup
 		// as the "previous path", for reference in case of a rename or relocation.
 		currPaths[cID] = currPath.String()
 	}
 
-	// any path that wasn't present in the resolver was deleted by the user.
-	// relocations and renames will have removed the dir by id earlier.  What's
-	// left in deletedPaths are only the previous paths that did not appear as
-	// children of the root.
-	for fID, dp := range deletedPaths {
+	// A tombstone is a collection path that needs to be marked for deletion.
+	// Tombstones can occur for a number of reasons: the delta token expired,
+	// the container was deleted in flight, or the user deleted the container
+	// between backup runs.  If events combine to both delete and write content
+	// to the same container (ex: container_1 gets deleted, then container_2
+	// gets created with the same name), it is assumed that the backup consumer
+	// processes deletions before creations, making the combined operation safe.
```

Comment:
this is a little unclear. Are we not the backup consumer?  This probably should be reworded. I would make a suggestion, but it is unclear. I would imagine that  container_1 gets deleted and container_2 is created with the same name, that container_2's `id` is different than the previous and would require that the `id` be updated. 
However, I don't think this is the operation that is being described at all.